api/src/main/java/com/staxpayments/api/requests/CreateInvoiceBody.kt

```diff
@@ -0,0 +1,16 @@
+package com.staxpayments.api.requests
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.JsonElement
+
+@Serializable
+data class CreateInvoiceBody(
```

Comment:
Let's change the naming convention to `InvoicePostRequest`. That way we know what the http route is based on the name.